packages/client/src/utils/html-utils.ts

```diff
@@ -23,3 +23,29 @@ export function createElementFromHTML(html: string): HTMLElement | undefined {
     }
     return undefined;
 }
+
+declare global {
+    interface HTMLElement {
+        next(): HTMLElement
+        previous(): HTMLElement
+        first(): HTMLElement
+        last(): HTMLElement
+    }
+}
+
+// HTMLElement extensions for readability and convenience (reduce casting)
+HTMLElement.prototype.next = function (): HTMLElement {
+    return this.nextElementSibling as HTMLElement;
+};
+
+HTMLElement.prototype.previous = function (): HTMLElement  {
+    return this.previousElementSibling as HTMLElement;
+};
+
+HTMLElement.prototype.first = function (): HTMLElement {
+    return this.firstElementChild as HTMLElement;
+};
+
+HTMLElement.prototype.last = function (): HTMLElement  {
+    return this.lastElementChild as HTMLElement;
+};
```

Comment:
It seems reasonable to me, but let us wait for what others will say.

Reply:
I used this a lot, so reusing this cast was tedious.